hack/jenkins/update-jobs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pass in the directory relative to /hack/jenkins, eg test-configs, in $1.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+jobs_dir=$1
+
+pushd hack/jenkins
+
+docker exec job-builder mkdir -p /${jobs_dir}
```

Comment:
We have nounset set so if nothing is provided in $1 it'll crash, but even so, probably safer to put it in /jobs or something.